### PR TITLE
[CDAP-16278] Modifies selection to be default in pipeline studio

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Modifies the behavior to default to selecting nodes while developing pipelines
- Provides a way to toggle between selection mode and move mode to move canvas around
- Fixes comments to not dissapear when copy/pasting

JIRA: https://issues.cask.co/browse/CDAP-16278
Build: https://builds.cask.co/browse/CDAP-UDUT531-1

On-hold: 
- Fixing e2e tests + adding new ones to validate the change. 
- Pipeline context menu changes

### CODE REVIEW

File: cdap-ui/app/directives/dag-plus/my-dag-ctrl.js [hunk 1]
```diff
@@ -72,15 +72,42 @@ angular.module(PKG.name + '.commons')
 
     var Mousetrap = window.CaskCommon.Mousetrap;
 
+    vm.clearSelectedNodes = () => {
+      console.trace();
```
Comment (elfenheart) [inline cdap-ui/app/directives/dag-plus/my-dag-ctrl.js]: nit: still needed?

**Review by elfenheart — COMMENTED** [review]
When i hold space bar and move down, the pipeline seemed to jumped up. Then when i release the mouse, it jumped back down to where it needs to be.

I also cannot select through the drag selection anymore

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Comment by ajainarayanan** [comment]
Green build: https://builds.cask.co/browse/CDAP-UDUT531-6

**Pull request merged**